OWNERS.md

```diff
@@ -1,31 +1,33 @@
 # Crossplane Maintainers
 
-This page lists all active maintainers and their areas of expertise. This can be used for routing PRs, questions, etc. to the right place.
+This page lists all active members of the steering committee and maintainers for **this**
+repository. Each repository in the [Crossplane organization](https://github.com/crossplane/) will
+list their repository maintainers in their own `OWNERS.md` file.
 
-* See [CONTRIBUTING.md](CONTRIBUTING.md) for general contribution guidelines.
-* See [GOVERNANCE.md](GOVERNANCE.md) for governance guidelines and maintainer responsibilities.
+Please see [GOVERNANCE.md](GOVERNANCE.md) for governance guidelines and responsibilities for the
+steering committee and maintainers.
 
-## Senior maintainers
-
-Here's the current list of senior maintainers:
+## Steering Committee
 
 * Bassam Tabbara <bassam@upbound.io> ([bassam](https://github.com/bassam))
-  * Catch-all, architecture, strategy
-* Jared Watts <jared@upbound.io> ([jbw976](https://github.com/jbw976))
-  * Stacks architecture, release engineering
+  * 2 year term expires `2022-05-xy`
```

Comment:
note these exact dates should be filled in before merging, 2 years from date of acceptance and merge